### PR TITLE
feat: enable launcher via linux app image

### PR DESCRIPTION
**Pull request by maybeast**

### CODE REVIEW

File: public/ipc.js [hunk 1]
```diff
@@ -18,14 +18,45 @@ const DOCKER_SETTINGS_PATH = path.join(
...
+    return `${LINUX_LAUNCHER_COMMAND} setup`;
+  }
+};
+
+const getStopOpendexDocker = () => {
+  if (os.platform() === "win32") {
+    return `set BRANCH=${BRANCH}&set NETWORK=${NETWORK}&${LAUNCHER} stop`;
+  } else {
+    // linux and mac
+    return `BRANCH=${BRANCH} NETWORK=${NETWORK} ${LAUNCHER} stop`;
```
Comment (stackingcats) [inline public/ipc.js:52]: Here we could also use the nice `*_LAUNCHER_COMMAND` variables.

File: src/common/dockerUtil.ts [hunk 2]
```diff
@@ -65,6 +66,28 @@ const isDockerInstalled$ = (): Observable<boolean> => {
...
+    map((output) => {
+      if (output.includes("docker-compose version")) {
+        return true;
+      }
+      return false;
+    }),
+    catchError((e: any) => {
+      if (
+        e.message?.includes(
+          "'docker' is not recognized as an internal or external command"
```
Comment (stackingcats) [inline src/common/dockerUtil.ts:80]: When executing `docker-compose version` on an Ubuntu machine without docker-compose, I get a response starting with `Command 'docker-compose' not found`.

File: src/common/dockerUtil.ts [hunk 3]
```diff
@@ -65,6 +66,28 @@ const isDockerInstalled$ = (): Observable<boolean> => {
...
+        )
+      ) {
+        return of(false);
+      }
+      logError("Error checking Docker installed status:", e);
```
Comment (stackingcats) [inline src/common/dockerUtil.ts:85]: This should probably refer to Docker-Compose, not Docker?

**Review by stackingcats — CHANGES_REQUESTED** [review]
Looks good in general. Just some minor things.
Also, let's handle the case when the user has not been added to docker group and gets `permission denied`.

**Comment by maybeast** [comment]
@stackingcats your feedback is addressed + `DockerPermissionDenied` screen was handled in order to point users to the https://docs.docker.com/engine/install/linux-postinstall/ docs.

File: src/setup/create/DockerPermissionDenied.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+import RowsContainer from "../RowsContainer";
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    buttonsContainer: {
+      minHeight: 90,
+    },
+    installLink: {
+      textDecoration: "none",
+    },
```
Comment (stackingcats) [inline src/setup/create/DockerPermissionDenied.tsx:26]:
```suggestion
    },
    stepsContainer: {
      width: "657px",
    },
```

File: src/setup/create/DockerPermissionDenied.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  }, [history]);
+
+  return (
+    <RowsContainer>
+      <Grid item container justify="center">
+        <Typography variant="h6" component="h2" align="center">
+          Docker permission denied. Please follow the post-installation steps
+          and log out and log back in.
+        </Typography>
+      </Grid>
```
Comment (stackingcats) [inline src/setup/create/DockerPermissionDenied.tsx:53]:
```suggestion
      <Grid item container direction="column">
        <Grid item container justify="center">
          <Typography variant="h6" component="h2" align="center">
            Docker permission denied. Please follow the post-installation steps
            and:
          </Typography>
        </Grid>
        <Grid item container justify="center">
          <ul className={classes.stepsContainer}>
            <li>Log out and log back in</li>
            <li>Restart your machine if still seeing this</li>
          </ul>
        </Grid>
      </Grid>
```

**Review by stackingcats — CHANGES_REQUESTED** [review]
Added two suggestions to improve the text in the `DockerPermissionDenied` screen.

**Review by stackingcats — APPROVED** [review]
LGTM :+1:

**Pull request merged**